plugins/Morpheus/stylesheets/theme-advanced.less

```diff
@@ -0,0 +1,17 @@
+@theme-color-background-base:          #fff;
+@theme-color-background-tinyContrast:  #f2f2f2;
+@theme-color-background-lowContrast:   @color-silver-l80;
+@theme-color-background-contrast:      #5F5A60;
+@theme-color-background-highContrast:  #202020;
+
+@theme-color-border: @color-silver-l80;
+
+@theme-color-menu-contrast-text: @theme-color-text-lighter;
```

Comment:
`@theme-color-menu-*` and `@theme-color-widget-*` should be in `theme.less` I think. Those are specifically exposed API's/variables to allow developers to change those. Or with other words those are the things we identified that companies might want to change if they want to adjust Piwik to their CI.